aiidalab/app.py

```diff
@@ -914,9 +924,12 @@ def _refresh_versions(self) -> None:
         )
 
     def _refresh_dependencies_to_install(self) -> None:
-        self.dependencies_to_install = self._app.find_dependencies_to_install(
-            self.version_to_install
-        )
+        if self.version_to_install:
```

Comment:
This could be None so mypy got rightfully mad that we're passing a possible None value to a function that did not expect it. In other words, this was a bug.